storehaus-postgresql/src/main/scala/com/twitter/storehaus/postgres/PostgresStore.scala

```diff
@@ -0,0 +1,123 @@
+package com.twitter.storehaus.postgres
+
+import com.twitter.storehaus.Store
+import com.twitter.util.{Future, Time}
+import roc.postgresql.{Client, Element, Request, Result}
+
+/**
+  * Key value storage with PostgreSQL (>9.5) as backend
+  *
+  * Created by ponkin on 9/24/16.
+  */
+object PostgresStore {
+
+  def apply[K: Show : Read, V: Show : Read](client: Client, table: String, kCol: String, vCol: String) =
+    new PostgresStore[K, V](client, table, kCol, vCol)
+}
+
+class PostgresStore[K: Show : Read, V: Show : Read]
```

Comment:
Well, that part turns out to be kind of tricky). How can we implement merge since we use async driver( separate calls client.query("BEGIN") and client.query("COMMIT;") can lead to unknown result, since some requests can be made simultaneously  and can be added in transaction). I do not know ho to tie Futures to particular connection. Any ideas? We can use optimistic locking but we need additional column to keep lock, like timestamp or version.